src/Knp/Menu/Integration/Symfony/RoutingExtension.php

```diff
@@ -11,6 +11,9 @@
  */
 class RoutingExtension implements ExtensionInterface
 {
+    /**
+     * @var UrlGeneratorInterface
```

Comment:
I don't think that's necessary. phpstan and PHPStorm are already able to infer it from the constructor

Reply:
same in other classes

Reply:
but maybe can be useful in the future when using rector? I'm thinking about raising minimum PHP version